integrations/tensorflow/e2e/keras/BUILD

```diff
@@ -0,0 +1,110 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load(
+    "//bindings/python:build_defs.oss.bzl",
+    "INTREE_TENSORFLOW_PY_DEPS",
+    "NUMPY_DEPS",
+)
+load(
+    "//build_tools/bazel:iree_py_test_suite.bzl",
+    "iree_py_test_suite",
+)
+load(
+    "//integrations/tensorflow/e2e/keras:iree_vision_test_suite.bzl",
+    "iree_vision_test_suite"
+)
+
+package(
+    default_visibility = ["//visibility:public"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+iree_py_test_suite(
+    name = "non_vision",
+    srcs = glob(
+        ["*_test.py"],
+        exclude = [
+            "keras_vision_model_test.py",
+            # TODO(GH-1620): Include after fixing the failure on GitHub Actions.
+            "keras_lstm_static_test.py"
+        ]
+    ),
+    deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+        "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+    ],
+)
+
+iree_vision_test_suite(
+    name = "keras_vision_models",
+    configurations = [
+        # tuples of (dataset, include_top, model_name, backends)
+        # "cifar10" has toy models with input 32x32, is good for debugging
+        ("cifar10", 1, "ResNet50", "tf,iree_vmla"),
+        ("cifar10", 1, "ResNet50", "tf,iree_llvmjit"),
+        ("cifar10", 1, "ResNet50", "tf,iree_vulkan"),
+    ],
+    deps = INTREE_TENSORFLOW_PY_DEPS + NUMPY_DEPS + [
+        "//integrations/tensorflow/bindings/python/pyiree/tf/support",
+    ],
+    tags = ["manual"],
+)
+
+iree_vision_test_suite(
+    name = "keras_vision_models_external",
+    configurations = [
+        # tuples of (dataset, include_top, model_name, backends)
+        # "cifar10" has toy models with input 32x32, is good for debugging
+        # "imagenet" has real model weights for input 224x224
+        ("cifar10", 1, "MobileNet", "tf,iree_vmla"),
+        ("cifar10", 1, "MobileNet", "tf,iree_vulkan"),  # TODO(b/150244105)
```

Comment:
This is what Oleg wrote. Here I am just refactoring the changes that he made.

Reply:
I could add something like a key to the top of this?

Reply:
How about adding to the top

`# TODO(b/150244105): Make all of these tests pass on all backends` or whatever it is exactly TODO here. Having the suite include some passing and some failing tests is going to make it a bit difficult to use, especially for periodic testing. We'll probably need to list them manually for that. Another option would be to add an additional parameter here like `known_failing` or `include_in_suite` so that the suite could be just the passing tests. That's going to get a bit close to hiding failing tests though....

Reply:
I added a description above. 

I'm not sure I follow why it would be more difficult to use. Don't we want to run both the successful and failing tests in our daily testing and upload the per-test results to a public facing doc so that we can track which ones are failing? Does a suite get in the way of that somehow?

Reply:
The high-level status will be "passing" or "failing". If there are some tests that are failing, then the status will always be "failing", which isn't very useful